target-app/Onboarding/Views/SignUpViewController.swift

```diff
@@ -0,0 +1,224 @@
+        
+        let overlayImage = UIImage(named: "sign-in-overlay")
+        overlayImageView.contentMode = UIView.ContentMode.scaleAspectFill
+        overlayImageView.frame.size.width = view.bounds.width
+        overlayImageView.frame.size.height = 190
+        overlayImageView.image = overlayImage
+        
+        lineView.translatesAutoresizingMaskIntoConstraints = false
+        lineView.layer.borderWidth = 1.0
+        lineView.layer.borderColor = UIColor.black.cgColor
+        
+        view.addSubviews(subviews: [
+            overlayImageView
+        ])
+        
+        setupViews()
+        setupLayout()
+        
+        stackView.addArrangedSubview(subviews: [
+            titleLabel,
+            nameLabel,
+            nameField,
+            emailLabel,
+            emailField,
+            passwordLabel,
+            passwordField,
+            confirmPasswordLabel,
+            confirmPasswordField,
+            genderLabel,
+            genderField,
+            signUpButton,
+            lineView,
+            signInButton
+        ])
+        
+        activateConstraints()
+    }
+    
+    private func setupViews() {
+        view.addSubview(scrollView)
+        scrollView.addSubview(contentView)
+        contentView.addSubview(stackView)
+    }
+    
+    private func setupLayout() {
+        NSLayoutConstraint.activate([
+            scrollView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
+            scrollView.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor),
+            scrollView.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor),
+            scrollView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor),
+            
+            contentView.topAnchor.constraint(equalTo: scrollView.topAnchor),
+            contentView.leadingAnchor.constraint(equalTo: scrollView.leadingAnchor),
+            contentView.trailingAnchor.constraint(equalTo: scrollView.trailingAnchor),
+            contentView.bottomAnchor.constraint(equalTo: scrollView.bottomAnchor),
+            contentView.widthAnchor.constraint(equalTo: scrollView.widthAnchor),
+            
+            stackView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 20),
+            stackView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 20),
+            stackView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -20),
+            stackView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor)
+        ])
+    }
+    
+    func activateConstraints() {
+        [titleLabel,
+         nameLabel,
+         nameField,
+         emailLabel,
+         emailField,
+         passwordLabel,
+         passwordField,
+         confirmPasswordLabel,
+         confirmPasswordField,
+         genderLabel,
+         genderField
+        ].forEach {
+            $0.attachHorizontally(
```

Comment:
There is no reason for attach those views (the included inside the `stackView`) to the `view` because they the already are taking the `stackView` width, you just have to attach the `stackView` to the view width and that's all. ie: 
```
stackView.attachHorizontally(
  to: view,
  leadingMargin: space,
  trailingMargin: space,
 )
```

Reply:
I tried it but it didn’t work. Not sure if its because I have a scrollView and a contentView around that.